pkg/ccl/importccl/import_into_test.go

```diff
@@ -147,11 +147,14 @@ func TestProtectedTimestampsDuringImportInto(t *testing.T) {
 
 	// Wait for the ranges to learn about the removed record and ensure that we
 	// can GC from the range soon.
-	gcRanRE := regexp.MustCompile("(?s)shouldQueue=true.*processing replica.*GC score after GC")
+	// This regex matches when all float priorities other than 0.00000.
+	matchNonZeroSmall := "0\\.\\d*[1-9]"       // matches 0.00123, 0.00001
+	matchNonZeroLarge := "[1-9]\\d*(\\.\\d+)?" // matches 123.000, 123.2, and 1234
+	nonZeroProgressRE := regexp.MustCompile(fmt.Sprintf("priority=(%s|%s)", matchNonZeroSmall, matchNonZeroLarge))
```

Comment:
I wanted to simplify this, but this was the best I could come up with given that Go's regex doesn't support negative lookahead... if any ideas come to mind, I'm open to suggestions. :)